src/afni_plugin.c

```diff
@@ -305,7 +305,6 @@ ENTRY("PLUG_get_many_plugins") ;
    if( epath == NULL )
      epath = getenv("AFNI_PLUGIN_PATH") ; /* try another name? */
 
-#ifdef WHATS_MY_EXEPATH
```

Comment:
@afni-rickr, removing this ifdef seems to cause a segfault. Any ideas why that would be?